python/src/KrigingResult_doc.i.in

```diff
@@ -292,6 +292,33 @@ for each point in sampleX."
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::KrigingResult::operator()
+"Compute the expected Gaussian distribution on a new point / sample conditionally to the observed paths.
+
+Available usages:
+    KrigingResult(pt)
+
+    KrigingResult(sampleX)
+
+Parameters
+----------
+pt : sequence of float
+    The point :math:`\vect{x}` where the conditional distribution of the output has to be evaluated.
+sampleX : 2-d sequence of float
+     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_M)` where the conditional distribution of the output has to be evaluated (*M* can be equal to 1).
+
+Returns
+-------
+condDist : :class:`~openturns.Normal`
+    The conditional Gaussian distribution.
+
+Notes
+-----
+The conditional distribution :math:`\cN(\Expect{\vect{Y}}, \Cov{\vect{Y}})` has respectively conditional mean and covariance functions 
+defined by :meth:`getConditionalMean` (respectively :meth:`getConditionalCovariance`)."
```

Comment:
"defined" is not really the correct word here, I think. We are talking about an implementation, not a definition.

Reply:
ok